### PR TITLE
adding string support

### PR DESCRIPTION
**Pull request by eitanz-coti**
adding string support

### CODE REVIEW

**Comment by smiller-coti** [comment]
[load_aes_key function](https://github.com/coti-io/coti-sdk-python/blob/b9fcbcfdc73a41d67038047853ef0bd4e735bf9f/coti/crypto_utils.py#L73): Since we deleted `write_aes_key`, should we consider deleting `load_aes_key`

[built_input_text function](https://github.com/coti-io/coti-sdk-python/blob/b9fcbcfdc73a41d67038047853ef0bd4e735bf9f/coti/crypto_utils.py#L124): I think we should rename this to `build_uint_input_text` to be consistent with the TypeScript SDK

**Review by smiller-coti — DISMISSED** [review]
Requested changes left in the previous comment

File: coti/utils.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def print_network_details(web3):
     print('latest block: ', web3.eth.get_block('latest').get('hash').hex())
 
 
-def print_account_details(web3):
+def print_account_details(web3): ##### OK ######
```
Comment (smiller-coti) [inline coti/utils.py]: Is it intentional to leave the `##### OK ######`?

File: coti/utils.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def load_contract(file_path):
         return file.read()
 
 
-def transfer_native(web3, recipient_address, private_key, amount_to_transfer_ether, native_gas_units):
+def transfer_native(web3, recipient_address, private_key, amount_to_transfer_ether, native_gas_units): ##### OK ######
```
Comment (smiller-coti) [inline coti/utils.py]: Is it intentional to leave the `##### OK ######`?

**Review by smiller-coti — COMMENTED** [review]
See previous comments

**Review by gmesika-coti — APPROVED** [review]

**Pull request closed without merging**